packages/client/src/App.js

```diff
@@ -1,29 +1,38 @@
 import React, { } from 'react';
 import {
-  BrowserRouter as Router,
-  Switch,
-  Route,
+  BrowserRouter as Router, Switch, Route
 } from 'react-router-dom';
 import './App.css';
 import HomePage from './pages/Home';
 import ProfilePage from './pages/Profile';
 import SignInPage from './pages/SignIn';
+import Signin from './components/SignIn';
 import { AuthProvider } from './providers/AuthProvider';
 
 function App() {
   return (
     <AuthProvider fallback="loading">
       {/* Add Navbar component on this line */}
       <Router>
-        <Switch>
-          <Route path="/" exact>
-            <HomePage />
-          </Route>
-          <Route path="/profile">
-            <ProfilePage />
-          </Route>
-          <Route path="/signin"><SignInPage/></Route>
-        </Switch>
+        <div className="auth-wrapper">
+          <div className="auth-inner">
```

Comment:
Whats the purpose of these two divs. They seem to be modifying the home page styling. 
<img width="1150" alt="Screen Shot 2020-07-17 at 9 14 09 PM" src="https://user-images.githubusercontent.com/8944270/87842490-769c4080-c872-11ea-85ca-ed43c1071f3e.png">

Reply:
Whoops. Added to specifically the SignIn component as it creates a shadow effect.